mlflow/utils/file_utils.py

```diff
@@ -694,81 +697,54 @@ def parallelized_download_file_using_http_uri(
     Returns a dict of chunk index : exception, if one was thrown for that index.
     """
 
-    def run_download(range_start, range_end):
-        template = """
+    def run_download(chunk: _Chunk):
+        try:
+            subprocess.run(
+                [
+                    sys.executable,
+                    download_cloud_file_chunk.__file__,
+                    "--range-start",
+                    str(chunk.start),
+                    "--range-end",
+                    str(chunk.end),
+                    "--headers",
+                    json.dumps(headers or {}),
+                    "--download-path",
+                    download_path,
+                    "--http-uri",
+                    http_uri,
+                ],
+                text=True,
+                check=True,
```

Comment:
Ah, I missed that we had added this in https://github.com/mlflow/mlflow/commit/460907be7b7cedef55f0c42cf2a8ef2f35d9755d#diff-6f28a867435ec4c1fc06706110a3b515fe096be3af1fb20ab70f3db5315f828f. Sorry for the confusion